.cirrus.yml

```diff
@@ -0,0 +1,77 @@
+# Reference:
+#   - https://cirrus-ci.org/guide/writing-tasks/
+#   - https://cirrus-ci.org/guide/linux/
+#   - https://hub.docker.com/_/gcc/
+#   - https://hub.docker.com/_/python/
+
+#
+# Global defaults.
+#
+container:
+  image: gcc:latest
+  cpu: 2
+  memory: 4G
+
+
+env:
+  # Maximum cache period (in weeks) before forcing a new cache upload.
+  CACHE_PERIOD: "2"
+  # Increment the build number to force new mambaforge cache upload.
+  MAMBA_CACHE_BUILD: "0"
+  # Increment the build number to force new cf-units cache upload.
+  CFUNITS_CACHE_BUILD: "0"
+  # Base environment conda packages to be installed
+  MAMBA_CACHE_PACKAGES: "pip conda-lock"
+  # Increment the build number to force new pip cache upload.
+  PIP_CACHE_BUILD: "0"
+
+
+#
+# Testing (Linux)
+#
+linux_task:
+  auto_cancellation: true
+  matrix:
+    env:
+      PY_VER: "3.7"
+    env:
+      PY_VER: "3.8"
+    env:
+      PY_VER: "3.9"
+      COVERAGE: "codecov"
+  name: "${CIRRUS_OS}: py${PY_VER}"
+  env:
+    PATH: ${HOME}/mambaforge/bin:${PATH}
+  mamba_cache:
+    folder: ${HOME}/mambaforge
+    fingerprint_script:
+      - echo "${CIRRUS_OS} $(sha256sum mambaforge.sh)"
+      - echo "${MAMBA_CACHE_PACKAGES}"
+      - echo "$(date +%Y).$(expr $(date +%U) / ${CACHE_PERIOD}):${MAMBA_CACHE_BUILD}"
+    populate_script:
+      - wget --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh -O mambaforge.sh
```

Comment:
Sorry, my fault totally... we need to revert this change and include the `wget` as part of the `fingerprint_script` as line 48 is performing a `sha256sum mambaforge.sh` of the downloaded file, sorry.

Reply:
https://github.com/SciTools/cf-units/pull/175#discussion_r651028077

Reply:
https://github.com/SciTools/cf-units/pull/175/commits/51a82523f0efe1ea2af1f9588395c61224a3980f